source/android/mobile/src/main/java/io/adaptivecards/adaptivecardssample/MainActivityAdaptiveCardsSample.java

```diff
@@ -367,7 +411,16 @@ private void renderAdaptiveCard(boolean showErrorToast)
 
             registerCustomFeatures();
 
-            RenderedAdaptiveCard renderedCard = AdaptiveCardRenderer.getInstance().render(this, getSupportFragmentManager(), parseResult.GetAdaptiveCard(), this, hostConfig);
+            RenderedAdaptiveCard renderedCard = AdaptiveCardRenderer.getInstance().render(
+                this,
+                getSupportFragmentManager(),
+                parseResult.GetAdaptiveCard(),
+                this,
+                null,
+                this,
+                null,
+                getActivityResultRegistry(),
```

Comment:
as this is the sample app, could you provide parameters that exercise our code rather than passing nulls?

Reply:
I have already added new variables for channelAdaptor and activityresultregistry. The null variable is part of the existing OverFlowMenuRenderer.